.travis.yml

```diff
@@ -5,7 +5,7 @@ dist: trusty
 addons:
   apt:
     sources:
-      - chef-current-trusty
```

Comment:
ditto above, a package was missing or something... this is what was required to get it to build.